python/activator/activator.py

```diff
@@ -50,8 +51,15 @@
 timeout = os.environ.get("IMAGE_TIMEOUT", 50)
 # Absolute path on this worker's system where local repos may be created
 local_repos = os.environ.get("LOCAL_REPOS", "/tmp")
-
-setup_google_logger(
+# Kafka server
+kafka_cluster = os.environ["KAFKA_CLUSTER"]
+# Kafka group; must be worker-unique to keep workers from "stealing" messages for others.
+kafka_group_id = str(uuid.uuid4())
```

Comment:
Why `uuid4` and not `uuid1`?

Reply:
Because the [uuid docs](https://docs.python.org/3/library/uuid.html) warned that `uuid1` has weaker concurrency guarantees, and that would make a difference if e.g. we ever have multiple workers per container. I really want to avoid the scenario we had on GCP where we tried to disambiguate on the PID, but every container always started a worker with PID 503.

Is there a reason `uuid1` would be preferable to `uuid4`?

Reply:
Butler uses uuid4.

Reply:
If we had a worker per IP, then uuid1 might give some debuggability.  But that's probably available elsewhere.  No big deal.